frontend/src/components/UserDashboard/UserDashboard.tsx

```diff
@@ -104,8 +108,8 @@ export const _UserDashboard = ({
             </Stack>
           ) : (
             <div className="grid grid-cols-2 text-center my-4">
-              <div>Learned 20 words</div>
-              <div>Learned 5 lessons</div>
+              <div>Learned {learnedWordsNum} words</div>
+              <div>Learned {learnedQuizzesNum} quizzes</div>
```

Comment:
If the use learns 1 quiz, does it display `1 quizzes` ?
You can use this [Laravel string helper](https://laravel.com/docs/8.x/helpers#method-str-plural)

Reply:
done